### DISCUSSION TITLE
Issue with Date Format in Date Range Picker

### DISCUSSION THREAD

**Discussion by kinjal1298** [malzariey/filament-daterangepicker-filter#103] [Q&A]
Hi everyone,

I'm currently facing an issue with a date range picker in my project. I need to display dates in the MM/DD/YYYY format for both the start date and end date. However, I'm encountering a problem where the start date is displayed in DD/MM/YYYY format, while the end date is correctly shown in MM/DD/YYYY format.

Additionally, when I load the page for the first time and open the dropdown calendar, it shows NaN instead of the actual dates. This makes it impossible to select a date range.

Here is a my current initialization code:
`DateRangePicker::make('date_range')
->label('Date Range')
->startDate(Carbon::now()->subDays(6))
->endDate(Carbon::now())
->maxDate(Carbon::now())
->maxSpan(['weeks' => 1])
->disableRanges()
->displayFormat("MM/DD/YYYY")
->format('m/d/Y'),`

If anyone knows please help

**Comment by malzariey** [comment]
Is the issue still there.

**Comment by malzariey** [comment]
If the issue presists please open an issue with a reproduction repository
